domains/thisaintnishant.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ThisAintNishant",
+        "email": "gangurdenishant07@gmail.com"
+    },
+    "record": {
+        "URL": "https://thisaintnishant.vercel.app/"
```

Comment:
You should complete your website first, i only see a text "Portfolio" "Will be Live Soon"

Reply:
ok i will complete soon
and push it